### PR TITLE
Only accept valid booleans in query params

### PR DESCRIPTION
**Pull request by kelunik**
- Accept `0` / `1` / `true` / `false` (case insensitive)
- Add tests for `RequestUtil` and `Paginated`
- Format `Paginated` according to the project's code style

### CODE REVIEW

**Comment by kelunik** [comment]
Coverage increased by 3.1%. \o/

**Comment by LeanderK** [comment]
yeah!

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java [hunk 1]
```diff
@@ -5,6 +5,10 @@
 import spark.Request;
 
 public class RequestUtil {
+    private RequestUtil() {
+        // utility class
+    }
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java:10]: why?

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java [hunk 2]
```diff
@@ -40,6 +44,15 @@ public static boolean getQueryBool(Request request, String param, boolean defaul
...
-        return Boolean.parseBoolean(paramValue);
+        switch(paramValue.toLowerCase()) {
+            case "1":
+            case "true":
+                return true;
+            case "0":
+            case "false":
+                return false;
+            default:
+                throw new BadRequestException("Query parameter '%s' must be a valid boolean, '%s' given. Use true / 1 / false / 0.", param, paramValue);
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java]: i would just duplicate the statements...its easier to read

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java (hunk 1 above)
Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java:10]: Convention to add private constructors to utility classes.

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java (hunk 2 above)
Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java]: I think it's harder to get that there are only two values, but I don't really care. Changed.

**Pull request merged**